packages/jetpack-ai-calypso/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@automattic/jetpack-ai-calypso",
-	"version": "1.0.0",
+	"version": "2.0.0",
 	"description": "Jetpack AI utilities available to Calypso.",
 	"calypso:src": "src/index.js",
```

Comment:
This points to `src/index.js`, but the real file name is `src/index.ts`. I suspect that this is causing the issues that forced you to add `SKIP_TSC=true` and remove the `yarn run clean` calls.

Reply:
Good catch! Thank you.